src/components/HealthMarkers.tsx

```diff
@@ -0,0 +1,53 @@
+// maps over 'healthMarkers' array
+// 1. checks 'type' to render icon and heading
+// 2. renders 'MarkerList' component, which will render individual cards
+
+// had to reformat data a little bit to optimize. 'healthMarkers' is an array of objects. Each object has a 'type' key that allows us to map all 3. The individual 'markers' data within each type is another array that we can esaily map through to render data.
+
+'use client'
+import healthData from '@/app/cohort_template'
+import styles from '../app/results/page.module.css'
+import MarkerList from './MarkerList'
+import { FaHeartbeat } from 'react-icons/fa'
+import { FaDumbbell } from 'react-icons/fa6'
+import { TbStretching } from 'react-icons/tb'
+
+const HealthMarkers = () => {
+  const { healthMarkers } = healthData['female_40_59']
```

Comment:
Separating this data out here is awesome! I am adding some code in a PR that will specify the age range so we can refactor from hardcoded to using that variable ✅